amti/actions/review.py

```diff
@@ -126,6 +154,9 @@ def review_batch(
         batch_dir, batchid_file_name)
     incomplete_file_path = os.path.join(
         batch_dir, settings.INCOMPLETE_FILE_NAME)
+    if mark_file:
+        marked_file_path = os.path.join(
+            batch_dir, mark_file+".json")
```

Comment:
We probably don't need to set the file extension anymore or place the file in the batch directory. Deleting these lines will allow the user to place the mark file wherever they like.

Reply:
Deleted